notes/returned-mail.vbs

```diff
@@ -128,7 +127,7 @@ DO
 			IF err_msg <> "" THEN MsgBox "*** NOTICE!***" & vbNewLine & err_msg & vbNewLine
 		End If
 	Loop until err_msg = ""
-	call back_to_self ' this is for if the worker has used the POLI/TEMP navigation'
+	CALL back_to_self ' this is for if the worker has used the POLI/TEMP navigation'
```

Comment:
There is no If...statement around calling back to self. I would take this outside of the do...loop and add it before the convert_date_into_MAXIS_footer_month(date_received, MAXIS_footer_month, MAXIS_footer_year) function on Line 135

Reply:
this is for if the worker has used the POLI/TEMP navigation - added to the IF statement to support that, CL and I had testing around this and we need it if the button is pressed that was the reason for the placement.